webxr/openxr/input.rs

```diff
@@ -96,6 +104,26 @@ impl ClickState {
         };
         (click.is_active, select_event)
     }
+
+    fn update_from_value(&mut self, value: bool, menu_selected: bool) -> Option<SelectEvent> {
+        let select_event = match (value, *self) {
```

Comment:
Can we avoid duplicating this code by replacing the block in update_from_action with a call to update_from_value?